klifs_utils/local/coordinates/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+klifs_utils
+Utility functions to work with KLIFS data (local)
+
+Coordinates.
+"""
+
+# Add imports of sub-modules here
+from .complex import mol2_to_dataframe
+from .protein import mol2_to_dataframe
+from .pocket import mol2_to_dataframe
+from .ligand import mol2_to_dataframe, mol2_to_rdkit_mol
+from .water import mol2_to_dataframe
```

Comment:
You are overriding `mol2_to_dataframe` in each line. Is this what you want to do?

Reply:
No! Thanks for pointing this out. 

I guess I do not need any of these lines because all I want is to be able to call

```python
protein.mol2_to_dataframe
pocket.mol2_to_dataframe
```